x-pack/legacy/plugins/code/public/reducers/repository_management.ts

```diff
@@ -93,18 +94,30 @@ export const repositoryManagement = handleActions<
         draft.importLoading = false;
         draft.showToast = true;
         draft.toastType = ToastType.success;
-        draft.toastMessage = `${action.payload!.name} has been successfully submitted!`;
+        draft.toastMessage = i18n.translate(
+          'xpack.code.repositoryManagement.repoSubmittedMessage',
+          {
+            defaultMessage: '{name} has been successfully submitted!',
+            values: { name: action.payload!.name },
+          }
+        );
         draft.repositories = [...state.repositories, action.payload!];
       }),
     [String(importRepoFailed)]: (state, action: Action<any>) =>
       produce<RepositoryManagementState>(state, draft => {
         if (action.payload) {
           if (action.payload.res.status === 304) {
-            draft.toastMessage = 'This Repository has already been imported!';
+            draft.toastMessage = i18n.translate(
+              'xpack.code.repositoryManagement.repoImportedMessage',
+              {
+                defaultMessage: 'This Repository has already been imported!',
+              }
+            );
             draft.showToast = true;
             draft.toastType = ToastType.warning;
             draft.importLoading = false;
           } else {
+            // TODO add localication for those messages
```

Comment:
yes, do we need to translate the messages from the backend? are there already some util functions to achieve this?

Reply:
If these messages are being shown to users then yes. Simply using `i18n.translate` for sending error messages from APIs is good. What utils would be useful for you here?

Reply:
so this part is hard to translate right now because the message is directly returned from nodegit library who doesn't have localization.